Client/Frontend/Browser/Playlist/Cells/PlaylistCellRedacted.swift

```diff
@@ -83,8 +83,7 @@ private struct PlaylistCellRedactedView: View {
           .redacted(reason: details == nil ? .placeholder : [])
           .shimmer(details == nil)
       }
-      
-      Spacer()
+      .frame(maxHeight: .infinity, alignment: .top)
```

Comment:
Sorry made a mistake when I gave you snippet previously thinking it was a `VStack`, alignment needs to be changed from `.top` to `.leading`, currently this is visually bugged:

https://user-images.githubusercontent.com/529104/185681792-8ca4b431-7311-416c-9365-1045b6760576.mp4